core/sections/clause_7_core.adoc

```diff
@@ -504,12 +504,151 @@ NOTE: This schema can also be obtained from https://raw.githubusercontent.com/op
 [[sc_process_inputs]]
 ===== Process inputs
 
-*_Overview:_*
+====== Overview
 
 Each process input is a name/value pair that appears in the `inputs` section of
-an execute request.
+an execute request as show in the following example:
+
+```
+"inputs": {
+   "inputName1": <input value 1>,
+   "inputName2": <input value 2>,
+   etc.
+}
+```
+
+The actual name of each input is its identifier as specified by the input's definition in the <<sc_process_description,process description>>.
+
+====== Cardinality
+
+NOTE: The `maxOccurs` parameter is deprecated in this standard and will be removed in a subsequent version of this document.  See <<deprecated-maxOccurs>> for details.
+
+The cardinality on an input is specified using the `minOccurs` and `maxOccurs` parameters from the input's definition in the <<sc_process_description,process description>>.  The default values of `minOccurs` and `maxOccurs` are 1 indicating that a single input of the corresponding name must be specified in an execute request.  The following table covers the various combinations of `minOccurs` and `maxOccurs` values.
+
+[[cardinality_rules]]
+.Cardinality rules for process inputs.
+[cols="10,10,60,40",options="header"]
+|===
+|minOccurs |maxOccurs |Interpretation |Example
+|0 |0 |Not Allowed |
+|0 |1 |A single input value *may* be optionally specified. | "input": {value}
+|1 |1 |A single input value *must* be specified. | "input": {value}
+|1 |N |At least 1 input value *must* be specified.  +
+Regardless of the number of input values provided, those values *must* be encoded using an array. | "input": [{value}] OR +
+"input": [{value1},...,{valueN}]
+|M |N |All values provided *must* be encoded using an array. |"input": [{value1},...,{valueM}] OR +
+"input": [{value1},...,{valueM},...,{valueN}]
+|===
+
+NOTE: The tokens `{value}`, `{value1}`, `{valueN}` or `{valueM}` represent values of the type expected for the input according to its definition.  This can include array values.
+
+====== Interaction of `minOccurs`/`maxOccurs` and `minItems`/`maxItems`
+
+The `schema` member of the <<process-input-schema,input definition>> in the <<sc_process_description,process description>> defines the schema of a single instance of an input.  If the input happens to be an array then the `minItems` and `maxItems` properties may be used to define the limits of the array.  The use of `minItems` and `maxItems` in the definition of the input does not affect how `minOccurs` and `maxOccurs` are interpreted and the same <<cardinality_rules,cardinality rules>> apply.
```

Comment:
This section seems to describe how cardinality is represented currently (1.0).

This doesn't align however with the `schema` using only `minItems` and `maxItems` in the `imagesInput` example to indicate the full cardinality representation, not a single instance of it. 

Doesn't this cause even more ambiguity regarding #373?

Reply:
Yeah, we discussed in the SWG and decided not to deprecate minOccurs and maxOccurs since this action, as you say, causes more ambiguity.  As a result, I am updating this PR to clarify minOccurs and maxOccurs but not to deprecate them.

Reply:
Done ... I hope! ;)